c2rust-transpile/src/translator/structs.rs

```diff
@@ -125,8 +123,7 @@ impl<'a> Translation<'a> {
                             mk().angle_bracketed_args(vec![mk().lifetime("a")]),
                         ),
                     ];
-                    let ty = mk().path_ty(mk().abs_path(path));
-                    ty
+                    mk().path_ty(mk().abs_path(path))
```

Comment:
`clippy` warns on this, right?  The `ty` variable name also serves as documentation.